astroquery/heasarc/core.py

```diff
@@ -86,8 +87,10 @@ def query_mission_cols(self, mission, cache=True, get_query_payload=False,
         All other parameters have no effect
         """
 
-        response = self.query_region_async(position='0.0 0.0', mission=mission,
-                                           radius='361 degree', cache=cache,
+        response = self.query_region_async(position=coordinates.SkyCoord(0.0, 0.0, unit='deg'),
+                                           mission=mission,
+                                           radius='361 degree',
```

Comment:
is this a hack to survey the whole sky?  Is this what is done through the online interface, or just something that you've verified works?

Reply:
I was not the one to add this approach here. I did change the radius from 100000 to 361 in the previous PR.
The purpose of this is to get the list of possible columns. 
It was decided by the original author that this is the way to do it. It seems to work so far.

Reply:
oh, yes, you're right, my bad